### PR TITLE
Refresh network-digi-sarian-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-digi-sarian-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 1]
```diff
@@ -1,49 +1,539 @@
...
+<TabItem value="CentOS 7" label="CentOS 7">
+
+```bash
+yum install centreon-pack-network-digi-sarian-snmp
+```
+
+</TabItem>
+</Tabs>
+
+2. Whatever the license type (*online* or *offline*), install the **Digi Sarian** connector through
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:176]:
```suggestion
2. Whatever the license type (*online* or *offline*), install the **Digi Sarian SNMP** connector through
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 2]
```diff
@@ -1,49 +1,540 @@
...
+<TabItem value="CentOS 7" label="CentOS 7">
+
+```bash
+yum install centreon-pack-network-digi-sarian-snmp
+```
+
+</TabItem>
+</Tabs>
+
+2. Quel que soit le type de la licence (*online* ou *offline*), installez le connecteur **Digi Sarian**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:177]:
```suggestion
2. Quel que soit le type de la licence (*online* ou *offline*), installez le connecteur **Digi Sarian SNMP**
```

File: pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 3]
```diff
@@ -1,49 +1,539 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=48034b/s;;80;90; 'interface_name2#interface.traffic.in.bitspersecond'=79181b/s;;80;90; 'interface_name1#interface.traffic.out.bitspersecond'=60438b/s;;80;90; 'interface_name2#interface.traffic.out.bitspersecond'=1708b/s;;80;90;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:335]: place des valeurs?

File: pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 4]
```diff
@@ -1,49 +1,539 @@
...
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--interface='.*' \
+	--name \
+	--add-status \
+	--add-traffic \
+	--critical-status='' \
+	--warning-in-traffic='80' \
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:327]: pourcentage?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 5]
```diff
@@ -1,49 +1,540 @@
...
+| *interface_name2*#interface.traffic.out.bitspersecond | b/s   |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Configuration SNMP
+
+Afin de superviser votre **Digi Sarian** en SNMP, il est nécessaire de configurer l'agent sur l'équipement.
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:129]:
```suggestion
L'agent SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 6]
```diff
@@ -1,49 +1,540 @@
...
+
+### Flux réseau
+
+La communication doit être possible sur le port UDP 161 depuis le collecteur
+Centreon vers le serveur supervisé.
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:134]:
```suggestion
La communication doit être possible sur le port UDP 161 depuis le collecteur Centreon vers le serveur supervisé.
```

File: pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 7]
```diff
@@ -1,49 +1,539 @@
...
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your **Digi Sarian** equipment.
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:128]:
```suggestion
The SNMP agent must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.
```

File: pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 8]
```diff
@@ -1,49 +1,539 @@
...
+
+### Network flow
+
+The target resource must be reachable from the Centreon poller on the UDP/161
+SNMP port.
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:133]:
```suggestion
The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 9]
```diff
@@ -1,49 +1,539 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=48034b/s;;80;90; 'interface_name2#interface.traffic.in.bitspersecond'=79181b/s;;80;90; 'interface_name1#interface.traffic.out.bitspersecond'=60438b/s;;80;90; 'interface_name2#interface.traffic.out.bitspersecond'=1708b/s;;80;90;
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:335]:
```suggestion
OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=48034b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=79181b/s;80;90;;  'interface_name1#interface.traffic.out.bitspersecond'=60438b/s;80;90;;  'interface_name2#interface.traffic.out.bitspersecond'=1708b/s;80;90;;  
```

File: pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 10]
```diff
@@ -1,49 +1,538 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=48034b/s;;80;90; 'interface_name2#interface.traffic.in.bitspersecond'=79181b/s;;80;90; 'interface_name1#interface.traffic.out.bitspersecond'=60438b/s;;80;90; 'interface_name2#interface.traffic.out.bitspersecond'=1708b/s;;80;90;
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:334]:
```suggestion
OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=48034b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=79181b/s;80;90;;  'interface_name1#interface.traffic.out.bitspersecond'=60438b/s;80;90;;  'interface_name2#interface.traffic.out.bitspersecond'=1708b/s;80;90;;  
```

File: pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md [hunk 11]
```diff
@@ -1,49 +1,538 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=48034b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=79181b/s;80;90;;  'interface_name1#interface.traffic.out.bitspersecond'=60438b/s;80;90;;  'interface_name2#interface.traffic.out.bitspersecond'=1708b/s;80;90;;
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-digi-sarian-snmp.md:334]:
```suggestion
OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=20b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=20b/s;80;90;;  'interface_name1#interface.traffic.out.bitspersecond'=20b/s;80;90;;  'interface_name2#interface.traffic.out.bitspersecond'=20b/s;80;90;;  
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**